api/controllers/EventController.js

```diff
@@ -113,7 +113,7 @@ var EventController = {
             if (e) return res.status(500).json(e.message);
 
             res.json({events: [event]});
-        })
+        }).populate('author');
```

Comment:
Wait a sec.
I think we might also need to add this to 
EventController/getEvents

Reply:
ya! good catch!

Reply:
@Smao0875 fixed